packages/scripts/utils/config.js

```diff
@@ -234,20 +234,20 @@ function getWebpackEntryPoints( buildType ) {
 	 * @return {Object<string,string>} The list of entry points.
 	 */
 	return () => {
-		// 1. Uses the recommended command format that lists entry points as paths to JavaScript files.
+		const entryPoints = {};
+
+		// 1. Collect all entry points from from the CLI arguments.
 		//    Example: `wp-scripts build one.js two.js`.
-		if ( process.env.WP_ENTRY ) {
-			return buildType === 'script'
```

Comment:
This early return is preventing the parsing of `**/block.json` for additional entries. Since there is no flag to enable it, we should keep processing instead of bailing early.